Website/Packages/datax-pipeline/src/modules/flowDefinition/flowReducer.js

```diff
@@ -189,7 +190,7 @@ export default (state = INITIAL_FLOW_STATE, action) => {
         case Actions.FLOW_UPDATE_SELECTED_FUNCTION_INDEX:
             return Object.assign({}, state, { selectedFunctionIndex: action.payload });
 
-        case Actions.FLOW_UPDATE_QUERY:
+        case QueryActions.QUERY_UPDATE_QUERY:
```

Comment:
It looks like this is a bug where you are handling and saving the query state here in the flow reducer from datax-pipeline and the query reducer from datax-query packages. This also applies above (in this file) for QUERY_UPDATE_RESAMPLING_INPUT_DURATION

Reply:
updated